docs/openapi/components/schemas/common/IntentToImport.yml

```diff
@@ -27,15 +27,15 @@ properties:
     description: >-
       Organization that is declaring their intention for a product to be exported to a
       different location than where it was created.
-    type: object
+    $ref: ./Organization.yml
```

Comment:
I don't think we want to pull in the whole Organization object. We want to only include a subset of what we want to use. Example here: https://github.com/w3c-ccg/traceability-vocab/blob/intent-to-import/docs/openapi/components/schemas/credentials/MillTestReportCredential.yml#L99